kahuna/public/js/image/view.html

```diff
@@ -160,14 +160,21 @@
                    id="show-usage"
                    class="radio-list__circle"
                    name="show-usage"/>
+
             <label for="show-usage"
                    class="radio-list__label"
-                   ng:class="{'radio-list--selected': ! showMetadata}"
-                   ng:click="showMetadata = false"
+                   ng:class="
+                   {
+                    'radio-list--disabled': ctrl.usagesCount < 1,
+                    'radio-list--selected': ! showMetadata
+                   }"
+                   ng:click="(ctrl.usagesCount < 1) || (showMetadata = false)"
                    gr:track-click="Image panel tab"
                    gr:track-click-data="{ tab: 'Usage' }">
                 <div class="radio-list__selection-state"></div>
-                <div class="radio-list__label-value">Usage</div>
+                <div class="radio-list__label-value">
+                    Usages ({{ctrl.usagesCount || "None"}})
```

Comment:
I personally favour [not using integers as booleans](https://github.com/guardian/flexible-content/wiki/JavaScript-Style-Guide#syntax) for clarity. Could use a ternary operator instead if you agree, but happy for others to chip in on this coding style.